src/index.html

```diff
@@ -206,9 +206,41 @@ <h2 class="h2">Next event</h2>
       </svg>
     </section>
 
+    <section class="my-16 tablet:mx-4 px-4">
+      <div class="tablet:w-[720px] desktop:w-[824px] flex flex-col items-center mx-auto tablet:grid grid-cols-2 tablet:grid-rows-smaller desktop:grid-rows-larger tablet:gap-x-4 desktop:gap-x-6">
+        <h2 class="h2 mobile:mb-0 tablet:mb-2.5 tablet:self-end w-[300px] text-center tablet:text-left">Chat with fellow attendees</h2>
+        <img 
+          class="row-span-full mobile:w-[351px] mobile:h-[344px] desktop:w-[400px] desktop:h-[392px] mx-auto tablet:mx-0"
+          src="./assets/img/slack.webp"
+          alt="Slack logo"
+          width="184px"
+          height="180px"
+          loading="lazy"
+        >
```

Comment:
I think the width and height are causing a "Properly size images" Lighthouse CI error - maybe they don't match the actual image dimensions? Or maybe the Tailwind class names not matching the attributes could be causing it?